### DISCUSSION TITLE
Sophia-Script for Build 19044.1288

### DISCUSSION THREAD

**Discussion by lcrocce19** [farag2/Sophia-Script-for-Windows#311] [Q&A] [answered]
Hi! Cannot find the specific script for Windows 10 LTSC 21H2 (19044.1288). Any help?

Thank you
![winver](https://user-images.githubusercontent.com/75593145/146095281-ccf9555d-83a2-493e-8dde-97ea8f2e8726.PNG)

**Comment by farag2** [comment, marked as answer]
https://github.com/farag2/Sophia-Script-for-Windows#windows-10

So you need Sophia.Script.for.Windows.10.LTSC.2021.v5.12.8.zip 😸

**Comment by mportioli** [comment]
Hello,

I tested the same Sophia Script version recommended version (5.12.8) but I get an error because in reality the minimum build 1348 is checked. Is that correct?

Thank you
![image](https://github.com/farag2/Sophia-Script-for-Windows/assets/51445128/486f167e-f612-4bd9-bd7d-9cda423c0b8f)

**Comment by farag2** [comment]
Hi
The latest supported version is https://github.com/farag2/Sophia-Script-for-Windows/releases/latest

5.12.8 isn't supported any more and contains errors

**Reply by mportioli** [reply]
Thank you, but I'm using Windows 10 IoT Enterprise LTSC Version 21H2 (OS Build 19044.1288). Is this a version supported by the latest release of your script?

**Comment by farag2** [comment]
Formally, no. Just update your Windows and try to run script. If you get error, please provide a screenshot. I will make it work on your Windows.
